.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: go
 go:
-  - 1.7
-  - 1.6.3
-  - 1.5.4
+  - 1.10
```

Comment:
This gets interpreted as 1.1 (and thus fails) -- you probably want to use `1.10.x` anyhow (to get the latest minor release in the 1.10 series).

Same with `1.9.x`.

See also https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use.

Reply:
> This gets interpreted as 1.1 (and thus fails)...

Thanks.  Fixed with 99c66a0 -> 692abcb.